src/libraries/System.Net.NetworkInformation/src/System.Net.NetworkInformation.csproj

```diff
@@ -5,8 +5,12 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Linux;$(NetCoreAppCurrent)-Browser;$(NetCoreAppCurrent)-OSX;$(NetCoreAppCurrent)-iOS;$(NetCoreAppCurrent)-tvOS;$(NetCoreAppCurrent)-FreeBSD</TargetFrameworks>
     <Nullable>enable</Nullable>
+    <NoWarn>$(NoWarn);nullable</NoWarn>
```

Comment:
Please do not add this.  Why is it needed?  If it must be added for browser, it needs to be special-cased to TargetsBrowser.

Reply:
It's the same as mentioned in https://github.com/dotnet/runtime/pull/38999#discussion_r452410323 and will be fixed that PR.